modules/react-components/src/transfer/tranfer.tsx

```diff
@@ -67,7 +67,9 @@ export const TransferComponent: FunctionComponent<PropsWithChildren<TransferComp
                                                         handleListSearch={ handleUnelectedListSearch }
                                                         placeholder={ searchPlaceholder }
                                                     />
+                                                    <Segment className="transfer-list-segment">
                                                     { list }
```

Comment:
Should have an indentation for the child `{ list }` element for readability.

Reply:
Resolved with e7e4251